### ISSUE TITLE
Markup decorative typography as decorative

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4211]
We have used bullets and vertical lines in the middle of text to differentiate between items, for example:

<img width="697" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/54e8623a-60e8-46da-8d8d-5f5c07efec7c">
<img width="198" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/c7e3b411-9bec-4945-954c-bab93a2117f9">

These are read out to by a screen reader as "vertical line" or "bullet" respectively.

1. this is confusing.  While the presentative elements are being spoken, they are not be communicated in a meaningful way that aids comprehension or navigation.
2. if items need to be differentiated, they should be formatted as a list.

[WCAG 1.3.1](https://www.w3.org/TR/WCAG22/#info-and-relationships)
> Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.

**Comment by StephDriver** [comment]
from #4194 

- Sample 1, item 21. issue details are read out as "volume 10 bullet issue 1 bullet 2023" which is confusing - make the 'bullets' decorative? <a href="/issue/910/info/">Volume 10 • Issue 1 • 2023</a>
- Sample 1, item 25. "vertical line ISSN: 2053 1915 vertical line published by" - confusing, make 'vertical lines' decorative. If the dash is important for the ISSN then this needs to be forced to read out rather than ignored as punctuation (depending on punctuation settings of user).
- Sample 1, item 27. "vertical line" - make this decorative.
- Sample 2, item 64. "2024 bullet Volume 9" is confusing, mark the bullet as decorative.
- Sample 2, item 76. "Vertical line ISSN" the | is being read aloud in the footer area.
- Sample 5.1, item 5. Issue title appears as an H1. It also has the 'bullet' spoken between each part of the title content e.g. "volume eight bullet issue one bullet two thousand and twenty four"
- Sample 5.1, item 15. decorative bullets between article details are read aloud.
- Sample 5.2, item 6. bullet is read out aloud between volume number and publication year in carousel item.
- Sample 5.2, item 13. "volume 23 bullet 2024" -bullet should not be articulated.
- Sample 5.3, item 16. Volume 15 • Issue 1 • 2023 the bullets are read aloud.
- Sample 5.3, item 22. again bullets between volume and issue and date are read aloud
- Sample Other, item 16. should the 'bullet' in lists be read out? or marked as decorative?
